LayoutTests/fast/table/inline-table-margin-baseline.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+ <html>
```

Comment:
Nit: this file has an indentation of 1 space that seems unnecessary.

Reply:
Indentation should be fixed